tensorflow/java/src/main/java/org/tensorflow/types/Types.java

```diff
@@ -0,0 +1,37 @@
+package org.tensorflow.types;
```

Comment:
Can we add a `package-info.java` to this package as well? In particular, I'm hoping that anyone who may be confused about why there are seemingly two ways to specify TensorFlow types (`DataType` enum and `Types.TFType` implementations) would read it and come our clearly satisfied. (Or perhaps this documentation belongs with the `Types.TFType` interface, but a `package-info.java` would still be nice)